flint.ui/src/store/modules/point.js

```diff
@@ -258,43 +240,39 @@ export default {
         simulation_step[step] = x
       }
 
-      console.log('pool 1')
-      for (let step = 0; step < df_as_array.length; step++) {
-        console.log(pool_1[step])
-      }
       console.log(simulation_step)
       console.log(pool_1)
       console.log(pool_2)
       console.log(pool_3)
 
-      this.Point_config_pool_1 = pool_1
-      this.Point_config_pool_2 = pool_2
-      this.Point_config_pool_3 = pool_3
+      this.state.point.pool_1 = pool_1
+      this.state.point.pool_2 = pool_2
+      this.state.point.pool_3 = pool_3
 
-      commit('update_Point_config_pool_1', pool_1)
-      commit('update_Point_config_pool_2', pool_2)
-      commit('update_Point_config_pool_3', pool_3)
+      commit('update_pool_1', pool_1)
+      commit('update_pool_2', pool_2)
+      commit('update_pool_3', pool_3)
 
-      console.log('this.Point_config_pool_1')
-      console.log(this.Point_config_pool_1)
-      console.log('this.Point_config_pool_2')
-      console.log(this.Point_config_pool_2)
-      console.log('this.Point_config_pool_3')
-      console.log(this.Point_config_pool_3)
-    }
+      console.log('this.pool_1')
+      console.log(this.state.point.pool_1)
+      console.log('this.pool_2')
+      console.log(this.state.point.pool_2)
+      console.log('this.pool_3')
+      console.log(this.state.point.pool_3)
+    },
   },
   getters: {
-    received_data: (state) => {
-      return state.received_data
+    results: (state) => {
+      return state.point.results
     },
-    Point_config_pool_1: (state) => {
-      return state.Point_config_pool_1
+    pool_1: (state) => {
+      return this.state.point.pool_1
```

Comment:
ESLint detected this as an error. Should it be `state.point.pool_1` to use the input `(state)` rather than the global `this`?

Reply:
Fixed during cleanup! 🎉